models/usage.go

```diff
@@ -0,0 +1,244 @@
+package models
+
+import (
+	"errors"
+	"time"
+
+	"github.com/c2h5oh/datasize"
+
+	"github.com/jinzhu/gorm"
+)
+
+// DataUsageTier is a type of usage tier
+// which governs the price per gb ratio
+type DataUsageTier string
+
+// String returns the value of DataUsageTier as a string
+func (d DataUsageTier) String() string {
+	return string(d)
+}
+
+// PricePerGB returns the price per gb of a usage tier
+// rtc indicates whether or not its an RTC based paymetn
```

Comment:
where does this `rtc` variable show up?

Reply:
Good catch, accidentally left a comment from an earlier implementation, will remove.